From 585f9a561a07ec1cf5c20af37229667a3009362a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Luis=20Javier=20Merino=20Mor=C3=A1n?= <ninjalj@gmail.com>
Date: Tue, 25 May 2021 13:14:20 +0200
Subject: [PATCH] Fix alpha channel of scrollbar colors

Commit 72d604c6cc96e0c2439244382ac07c6601970835 refactored some code
from TerminalDisplay into a new TerminalColor class, and in the process
lost an F on setAlphaF().

BUG: 437223
(cherry picked from commit 1cda384ec34e70ce052f8d177a8afcb665964663)
---
 src/terminalDisplay/TerminalColor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/terminalDisplay/TerminalColor.cpp b/src/terminalDisplay/TerminalColor.cpp
index 79dc1cd2..042135a0 100644
--- a/src/terminalDisplay/TerminalColor.cpp
+++ b/src/terminalDisplay/TerminalColor.cpp
@@ -121,7 +121,7 @@ namespace Konsole
 
         QColor buttonTextColor = m_colorTable[DEFAULT_FORE_COLOR];
         QColor backgroundColor = m_colorTable[DEFAULT_BACK_COLOR];
-        backgroundColor.setAlpha(m_opacity);
+        backgroundColor.setAlphaF(m_opacity);
 
         QColor buttonColor = backgroundColor.toHsv();
         if (buttonColor.valueF() < 0.5) {
-- 
GitLab

